wagtail/admin/panels/field_panel.py

```diff
@@ -11,10 +11,14 @@
 
 class FieldPanel(Panel):
     TEMPLATE_VAR = "field_panel"
+    BASE_ATTRS = { "data-field-wrapper":True }
 
     def __init__(
-        self, field_name, widget=None, disable_comments=None, permission=None, **kwargs
+        self, field_name, widget=None, disable_comments=None, permission=None, attrs=None, **kwargs
     ):
+        self.attrs = self.BASE_ATTRS
+        if attrs is not None:
+            self.attrs.update(attrs)
```

Comment:
If we use the approach already set in `Panel`, we should only need one single line added here - the 
```python
BASE_ATTRS = { "data-field-wrapper":True }
```
on line 14, the pulling out of `attrs` will be handled without any additional code by the root class implementation.